### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Nahid1911**
### **Project requirements**

- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

**for more requirements please click [HERE](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md).**

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @Nahid1911,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly add header links for these sections (Install, Usage, and Deployment) in your readme for easier navigation and professionalism.

kdkdo
kkdld

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.css [hunk 1]
```diff
@@ -1,19 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
```
Comment (divinecharlotte) [inline src/index.css:4]: - [ ] kindly consider fixing your design by sticking to the given one as much as possible  to make your design look exactly as expected check on the image below to see the main parts I highlighted
Screen Shot 2023-06-22 at 18 19 49

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Nahid1911     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Good branch naming👍 
- Descriptive pr 👍 

 




 

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (Nahid1911) [reply src/index.css:4]: Hello @divinecharlotte 
Thank you for your review.
1. Font Family and small gray line resolved.
2. Border around button removed.
3. Font width is not mention in the Zeplin design.
4. Author to Categories, I would like to mention that in the first day project requirements mentioned as follows, 
image

please visit the link of [first day project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md)

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Nahid1911 

Your project is complete! Great job for implementing the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- styling well implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**